src/network/clientpackethandler.cpp

```diff
@@ -1397,20 +1397,24 @@ void Client::handleCommand_HudSetSky(NetworkPacket* pkt)
 		*pkt >> skybox.body_orbit_tilt;
 	}
 
-	if (pkt->getRemainingBytes() >= 6) {
-		*pkt >> skybox.fog_distance >> skybox.fog_start;
-	}
-
-	if (pkt->getRemainingBytes() >= 4) {
-		*pkt >> skybox.fog_color;
-	}
```

Comment:
old servers will still send fog information in SET_SKY and the client needs to continue handling it